services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -458,8 +491,8 @@ export async function setServiceImageTagState(
 
   const { enabled } = req.body;
 
-  await setEnabled(enabled);
-  res.statusCode = 200;
+  const statusCode = await setEnabled(enabled);
+  res.statusCode = statusCode;
   res.send({ 'enabled': enabled });
```

Comment:
If `statusCode` === 423 then this should probably send and error message instead of this JSON.

Reply:
update.